geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @fileoverview This file provides a "mymaps" directive. This directive is
+ * used to insert a MyMaps block  into the HTML page.
+ * Example:
+ *
+ * <app-mymaps></app-mymaps>
+ *
+ */
+goog.provide('app.MymapsDirectiveController');
+goog.provide('app.mymapsDirective');
+
+goog.require('app');
+
+
+/**
+ * @return {angular.Directive} The Directive Object Definition.
+ * @param {string} appMymapsTemplateUrl
+ * @ngInject
+ */
+app.mymapsDirective = function(appMymapsTemplateUrl) {
+  return {
+    restrict: 'E',
+    scope: true,
```

Comment:
You could have used `scope: {}` here, our application-specific directives all using isolate scopes. But this can be done later when you extend the directive.